src/lib/errors.ts

```diff
@@ -21,21 +19,6 @@ export class NoEnvironmentError extends Error {
   name = "NoEnvironmentError";
 }
 
-export class FailedYahooValidationError extends Error {
-  name = "FailedYahooValidationError";
-  result: any;
-  errors?: null | ErrorObject[];
-
-  constructor(
-    message: string,
-    { result, errors }: { result: any; errors?: null | ErrorObject[] }
-  ) {
-    super(message);
-    this.result = result;
-    this.errors = errors;
-  }
-}
-
```

Comment:
This one unfortunately we'll need to keep, otherwise it's a breaking change.

I don't actually mind if we return the typebox errors here (vs ajv), I realise that's still a breaking change but I'll bite the bullet on this one as I don't think anyone is really inspecting the errors closely (beyond the `message` string property, which we should retain), but, we should still export a `FailedYahooValidationError` class and use it so that users can match with `instanceof`.

Reply:
Not an issue, I've added it back with some light modification of the type to be an array of Typebox errors. As you say this is a breaking change, so up to you as to whether we do a major or minor bump. But from the perspective of an `instanceof` check nothing should have changed